.gitignore

```diff
@@ -1,3 +1,5 @@
+/.vscode
+/.idea
```

Comment:
I figured this was uncontroversial, though it could certainly be left out.

Reply:
generally this kind of thing goes in one’s global gitignore rather than needing to be added to every repo one touches

Reply:
Hmmm, I did know that existed and for whatever reason I've just never done that. Probably because I've always just seen project `.gitignore` files include these.

Removed.